packages/attestation-service/src/index.ts

```diff
@@ -33,23 +42,27 @@ async function init() {
 
   await initializeSmsProviders(deliveryStatusURLForProviderType)
 
-  const app = express()
-  app.use([requestIdMiddleware(), loggerMiddleware])
-  const port = process.env.PORT || 3000
-  app.listen(port, () => rootLogger.info({ port }, 'Attestation Service started'))
-
   const rateLimiter = rateLimit({
     windowMs: 5 * 60 * 100, // 5 minutes
     max: 50,
   })
+  const app = express()
+  app.use([requestIdMiddleware(), loggerMiddleware, rateLimiter])
```

Comment:
so this now limits all requests with the same ratelimiter?

Reply:
yep. I will raise this number however, as a load balancer might well ping every 10secs by itself